polaris-react/src/components/FullscreenBar/FullscreenBar.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../styles/common';
+
+.FullscreenBar {
+  position: relative;
```

Comment:
Might want to change this to fixed to match the current top bar

Reply:
I actually copied this from the current `TopBar.scss` that says:

```
.TopBar {
  position: relative;
  display: flex;
  height: top-bar-height();
  ...
```

But is there a different component I should reference?